components/scripts/lib/cli-parsers/common.m4

```diff
@@ -3,6 +3,7 @@
 # ARG_OPTIONAL_BOOLEAN([enable-gradle-enterprise],[e],[])
 # ARG_OPTIONAL_SINGLE([git-branch],[b],[])
 # ARG_OPTIONAL_SINGLE([git-commit-id],[c],[])
+# ARG_OPTIONAL_SINGLE([git-options],[o],[])
```

Comment:
We also need to print the options in the summary section.

<img width="706" alt="image" src="https://user-images.githubusercontent.com/231070/197940010-2b591848-f641-4c14-8734-f760e00d8354.png">